corehq/apps/app_manager/suite_xml/sections/sync_requests.py

```diff
@@ -121,10 +120,10 @@ def get_module_contributions(self, module):
                 stack=Stack(),
             )
 
-            frame = PushFrame()
+            frame = CreateFrame()
             # Open first form in module
             frame.add_command(XPath.string(id_strings.menu_id(module)))
-            frame.add_datum(StackDatum(id=CALCULATED_DATA, value=XPath.string(MARK_AS_CLAIMED)))
+            frame.add_datum(StackDatum(id='case_id', value=QuerySessionXPath('case_id').instance()))
```

Comment:
Yeah. We are still figuring out what the advanced module workflow is is going to be.